remoteappmanager/rest/registry.py

```diff
@@ -0,0 +1,52 @@
+from remoteappmanager.rest.resource import Resource
+
+
+class Registry:
+    def __init__(self):
+        self._registered_types = {}
+
+    def register(self, typ):
+        """Registers a Resource type with an appropriate
```

Comment:
But `register` has to be specifically called anyway... It is unlikely future data model would have `__collection_name__` so that's no big deal, just wondering.